logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/Message.kt

```diff
@@ -37,7 +37,7 @@ sealed class Message(
 
                 is MessageContent.Asset -> {
                     contentString = "content:{sizeInBytes:${content.value.sizeInBytes}," + "mimeType:${content.value.mimeType}," +
-                            "metaData : ${content.value.metadata}, downloadStatus: ${content.value.downloadStatus}}"
+                            "metaData : ${content.value.metadata}, downloadStatus: ${content.value.downloadStatus}, uploadStatus: ${content.value.uploadStatus}}"
```

Comment:
@MohammedMokresh do we need to redact anything here in the meta data?

Reply:
as from our discussion last time we will keep this metaData as it will be the width , height for the video and images
duration, loudness  for Audio

Reply:
But his `toString()` method already contained the `downloadStatus`, so I simply replicated what had been done before and added the `uploadedStatus` method. By the way, if we don't add the rest of the data to this method, debugging becomes more painful, as it will only print the values that we decide to add here.